src/components/GraphQLQueryBuilder/addSignatory.tsx

```diff
@@ -140,7 +140,7 @@ export default function AddSignatoryQueryBuilder() {
         <br />
         See examples of various{' '}
         <a
-          href="https://docs.criipto.com/getting-started/token-contents/"
+          href="https://docs.criipto.com/verify/reference/token-contents/"
```

Comment:
Oh, we might want to remember to remove all these `https://docs.criipto.com` prefixes.